libs/chains/package.json

```diff
@@ -9,7 +9,8 @@
   "main": "build/index",
   "types": "src/index",
   "scripts": {
-    "build": "tsc -b",
+    "build": "tsc --version ; tsc -b",
```

Comment:
do we need to keep --version here?

Reply:
No.. I"ll remove this. I was debugging something.